src/main/java/core/basesyntax/service/impl/ReportCreatorImpl.java

```diff
@@ -13,11 +12,11 @@ public class ReportCreatorImpl implements ReportCreator {
     public String createReport() {
         StringBuilder stringBuilder = new StringBuilder(HEADING)
                 .append(System.lineSeparator());
-        for (Map.Entry<Fruit, Integer> record : Storage.fruits.entrySet()) {
+        for (Map.Entry<String, Integer> record : Storage.fruits.entrySet()) {
             stringBuilder.append(record.getKey())
+                    .append(System.lineSeparator())
```

Comment:
In this case you will have next report:
HEADING (+ lineSeparator ->)
record.getKey (+ lineSeparator ->)
,record.getValue